lib/bugsnag/payload.ex

```diff
@@ -62,23 +61,30 @@ defmodule Bugsnag.Payload do
         %{
           file: "unknown",
           lineNumber: 0,
-          method: "#{ module }.#{ function }#{ format_args(args) }"
+          method: Exception.format_mfa(module, function, args)
         }
       ({ module, function, args, [file: file, line: line_number] }) ->
+        file = to_string file
         %{
-          file: file |> List.to_string,
+          file: file,
           lineNumber: line_number,
-          method: "#{ module }.#{ function }#{ format_args(args) }"
+          inProject: Regex.match?(~r/^(lib|web)/, file),
```

Comment:
I'm still iffy on this since I don't want to make too many assumptions about project structure. How about just make it configurable? This is seems like a good default but I could imagine people having different setups.